components/blitz/src/ome/formats/importer/transfers/AbstractExecFileTransfer.java

```diff
@@ -48,7 +48,7 @@
     /**
      * "Transfer" files by soft-linking them into place. This method is likely
      * re-usable for other general "linking" strategies by overriding
-     * {@link #createProcessBuilder(File, File)} and the other protected methods here.
+     * {@code createProcessBuilder(File, File)} and the other protected methods here.
```

Comment:
No, something else in the file was, then I noticed it was causing a warning in Eclipse.